.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,130 @@
+name: Continous Integration
+
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+        node-version: [12]
+        include:
+          - os: ubuntu-latest
+            node-version: 10
+          - os: ubuntu-latest
+            node-version: 14
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Update NPM
+        run: npm i -g npm
+      - name: Bootstrap project
+        run: |
+          npm ci --ignore-scripts
+          npm run postinstall
+        env:
+          CI: true
```

Comment:
Let's set this env variable for all build steps please. I think some of our tests are checking `process.env.CI` too.